iteration_test.go

```diff
@@ -146,6 +155,56 @@ func (rest *TestIterationREST) TestSuccessUpdateIteration() {
 	_, updated := test.UpdateIterationOK(t, svc.Context, svc, ctrl, itr.ID.String(), &payload)
 	assert.Equal(t, newName, *updated.Data.Attributes.Name)
 	assert.Equal(t, newDesc, *updated.Data.Attributes.Description)
+	require.NotNil(t, updated.Data.Relationships.Workitems.Meta)
+	assert.Equal(t, 0, updated.Data.Relationships.Workitems.Meta["total"])
+	assert.Equal(t, 0, updated.Data.Relationships.Workitems.Meta["closed"])
+}
+
+func (rest *TestIterationREST) TestSuccessUpdateIterationWithWICounts() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
+
+	itr := createSpaceAndIteration(t, rest.db)
+
+	newName := "Sprint 1001"
+	newDesc := "New Description"
+	payload := app.UpdateIterationPayload{
+		Data: &app.Iteration{
+			Attributes: &app.IterationAttributes{
+				Name:        &newName,
+				Description: &newDesc,
+			},
+			ID:   &itr.ID,
+			Type: iteration.APIStringTypeIteration,
+		},
+	}
+	// add WI to this iteration and test counts in the response of update iteration API
+	wirepo := workitem.NewWorkItemRepository(rest.DB)
+	for i := 0; i < 4; i++ {
+		wirepo.Create(
```

Comment:
Some success/failure assertion here would be good since you have a dependent code following this.